server/service/src/processors/transfer/invoice/mod.rs

```diff
@@ -271,6 +271,24 @@ fn get_delete_operation(
 
     Ok(Operation::Delete { linked_invoice })
 }
+#[derive(Debug)]
+#[allow(dead_code)] // The "Wrong" variants with fields don't use the fields yet. Be warned allowing dead_code here silences when we never use a particular variant as well.
+enum InvoiceTransferOutput {
```

Comment:
This is great ! Although as a patch (even though it's feature), would probably do it in two parts, one for the actual fix (2.9.4) and one for adding the error types (can do to 2.9.4 then to dev when that's merged)

Reply:
I think a bit too much work to de-couple, in general please